custom_components/powercalc/data/signify/929002966401/model.json

```diff
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plug S",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.3.13:docker"
+    },
+    "name": "Hue White and Color Ambiance Surimu (square)",
+    "standby_power": 0.39
```

Comment:
you can use `aliases` here to add `929002966501` as an alias to eliminate copy/pasting the profile.
See some other model.json for example. See the supported model listing for models which have aliases defined.

Reply:
Thanks, done :)

Reply:
Seems you forgot to add the alias yet. it's not in your commit changeset

Reply:
I see, i have some issues with git, it is on my local repo and it is pushed but not visible in github...

Reply:
2nd commit worked :)